speedy-recovery-client/src/service/FhirMappingAdapterSTU3.js

```diff
@@ -16,7 +18,7 @@ const mapPatientToUserSTU3 = fhirPatientResource => ({
   id: fhirPatientResource.id ? fhirPatientResource.id : missingField,
   //This is a temporary hard-code fix as we have not implemented the searching for a patients' parent
   role:
-    fhirPatientResource.id === "f0462936-eb4b-4da1-b45a-fbd96ebf8ccb"
```

Comment:
Why has this change happened? The new value is invalid?

Reply:
We use kawai's script to add the parent resource(has a link to the patient), the new ID is this one, but the sanbox clear the added data everyday, so the value is invalid now, we need to update it.